tools/pipelines/scripts/build-version.js

```diff
@@ -3,6 +3,12 @@
  * Licensed under the MIT License.
  */
 
+"use strict";
```

Comment:
note these changes won't be merged. They're included to double-check that the new JS is functionally equivalent to the old divergent file.